client/my-sites/stats/stats-period-header/index.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import { PureComponent } from 'react';
+import './style.scss';
+
+class StatsPeriodHeader extends PureComponent {
```

Comment:
This looks like a small component. Do you think it could be converted to a function component instead of a class?

Reply:
Good catch!

Reply:
Resolved in [12ace47](https://github.com/Automattic/wp-calypso/pull/69661/commits/12ace47e758e40c6a4630d7b5ddd29afbc109e8a).